### PR TITLE
Philipp lig 1200 clean up docs first steps

### PR DESCRIPTION
**Pull request by philippmwirth**
# Philipp lig 1200 clean up docs first steps

It's best to quickly build the docs with `make html-noplot` for reviewing this.

Review and merge #820 first.

### CODE REVIEW

File: docs/source/docker/getting_started/first_steps.rst [hunk 1]
```diff
@@ -462,34 +444,7 @@ an automatically generated PDF report.
 Sample reports can be found on the `Lightly website <https://lightly.ai/analytics>`_.
 
 
-.. _ref-docker-runs:
-
-Live View of Docker Status
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:467]: Where did we move this section to? I think it's still useful to have the option of a "live view"

File: docs/source/docker/getting_started/first_steps.rst [hunk 2]
```diff
@@ -3,14 +3,13 @@
 First Steps
 ===================================
 
-The Lightly Docker solution follows a train, embed, select flow using 
-self-supervised learning.
+The Lightly Docker solution follows a train, embed, enhance, select workflow:
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:6]: Lightly Worker

File: docs/source/docker/getting_started/first_steps.rst [hunk 3]
```diff
@@ -20,202 +19,237 @@ self-supervised learning.
...
+        lightly/worker:latest \
+        token=MY_AWESOME_TOKEN \
+        worker.worker_id=MY_WORKER_ID
 
-Storage Access
------------------------------------
 
-We use volume mapping provided by the docker run command to process datasets. 
+Here, we use volume mapping provided by the docker run command to process datasets. 
 A docker container itself is not considered to be a good place to store your data.
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:50]: Lightly Worker

File: docs/source/docker/getting_started/first_steps.rst [hunk 4]
```diff
@@ -20,202 +19,237 @@ self-supervised learning.
...
+The docker solution can easily be triggered from your Python code. There are various parameters you can configure and we put a lot of effort to also expose the full lightly framework configuration.
 You could use the docker solution to train a self-supervised model instead of using the Python framework.
 
-Before jumping into the detail let's have a look at some basics.
-The docker container can be used as a simple script. You can control parameters by changing flags.
+Volume Mapping
+--------------
 
-Use the following command to get an overview of the available parameters:
+Before we jump into the details of how to submit jobs, we need to start the Lightly docker container in
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:37]: Lightly Worker

File: docs/source/docker/getting_started/first_steps.rst [hunk 5]
```diff
@@ -20,202 +19,237 @@ self-supervised learning.
...
    analytics in form of a pdf report with plots.
 
-You can use each of the three steps independently as well. E.g. you can use
-the Lightly Docker to embed a dataset and train a linear classifier on top of 
-them.
-
 
-The docker solution can be used as a command-line interface. You run the container, tell it where to find data, and where to store the result. That's it.
-There are various parameters you can pass to the container. We put a lot of effort to also expose the full lightly framework configuration.
+The docker solution can easily be triggered from your Python code. There are various parameters you can configure and we put a lot of effort to also expose the full lightly framework configuration.
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:31]: Lightly Worker

File: docs/source/docker/getting_started/first_steps.rst [hunk 6]
```diff
@@ -225,230 +259,178 @@ The command below will **train a self-supervised model** for (default: 100)
...
+                "score_name": "uncertainty_margin"
+            },
+            "object_level": {
+                "task_name": ""
+            }
+        }
+    )
+
+You may not always want to train for exactly 100 epochs with the default settings.
+The docker container is a wrapper around the lightly Python package.
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:294]: Lightly Worker

**Review by IgorSusmelj — CHANGES_REQUESTED** [review]
Looks great. Just added a few small things!

File: docs/source/docker/getting_started/first_steps.rst [hunk 7]
```diff
@@ -20,202 +19,237 @@ self-supervised learning.
...
+            ## Google Cloud Storage
+            # Input bucket
+            client.set_gcs_config(
+                resource_path="gs://bucket/input/",
+                project_id="PROJECT-ID",
+                credentials=json.dumps(json.load(open('credentials_read.json'))),
+                thumbnail_suffix=".lightly/thumbnails/[filename]_thumb.[extension]",
+                purpose=DatasourcePurpose.INPUT
+            )
+            # Output bucket
```
[the remark this answers is not in the document]
Comment (philippmwirth) [reply docs/source/docker/getting_started/first_steps.rst:117]: I thought you wanted to re-introduce these terms? I would leave it like this.

File: docs/source/docker/getting_started/first_steps.rst (hunk 1 above)
Comment (philippmwirth) [reply docs/source/docker/getting_started/first_steps.rst:467]: It's explained under "Scheduling a Simple Job"

File: docs/source/docker/getting_started/first_steps.rst [hunk 8]
```diff
@@ -20,437 +19,429 @@ self-supervised learning.
...
 
-Then you can add `relevant_filenames_file='subdir/relevant_filenames.txt'`
-to the docker run command and the Lightly docker will only consider **my-video.mp4** and **my-third-video.mp4**.
+.. code-block:: python
+    :caption: Creating a new dataset from Python
 
-Embedding a Dataset and Selecting from it
------------------------------------------
+    from lightly.api import ApiWorkflowClient
+    from lightly.openapi_generated.swagger_client.models.datasource_purpose import
```
Comment (IgorSusmelj) [inline docs/source/docker/getting_started/first_steps.rst:76]: typo?

**Review by IgorSusmelj — APPROVED** [review]
LGTM

**Pull request merged**